content/posts/manual-triggering.md

```diff
@@ -0,0 +1,104 @@
+---
+title: (Tests) job triggering improvements
+date: 2023-07-11T08:46:18+02:00
+weight: 88
+---
+
+Recently, we received multiple contributions from the Strimzi team, specifically [Jakub Stejskal](https://github.com/Frawless)
+and [David Kornel](https://github.com/kornys),
+for the functionality of manual triggering of jobs and other related improvements, mostly focusing on the testing
+UX. We are very happy about these and would like to showcase the results of their awesome contributions.
+
+Strimzi team wanted to onboard Packit to ease their testing in pull requests, but they have had multiple test suites that could run
+for a long time, and they did not want to run all of the test suites for each new commit in a pull request.
+
+Let's now look together at what they implemented to solve their usecase.
+
+## Manual-only triggering of jobs
+
+Firstly, a new configuration option [`manual_trigger`](/docs/configuration/jobs#manual_trigger) was introduced.
+With this new configuration option, users can enable triggering Packit jobs only manually and avoid specified jobs being
+automatically triggered when, e.g., a new commit arrives to a pull request.
+The only thing needed to make this work is to add `manual_trigger: true` to the job's definition:
+
+```yaml
+- job: tests
+  trigger: pull_request
+  targets:
+    - centos-stream-9-x86_64
+  skip_build: true
+  manual_trigger: true
+```
+
+This new configuration option allows saving resources and running builds or tests, e.g. only when the pull request is ready for
+testing and can be especially useful for projects having huge test plans.
+
+## Triggering test jobs based on labels and identifiers
+
+The above solution is very easy to use; however, there might be use cases where the users don’t want to trigger all the jobs.
+A particular example could be running a smoke test job before running the sanity one.
+
+To solve this, for the test jobs, guys from Strimzi introduced 2 new ways to trigger a specific job.
+
+The first one is to trigger the job based on the configured [`identifier`](/docs/configuration/upstream/tests#optional-parameters). To trigger a test job with `identifier: test-1` in the
+job configuration, the Packit comment command will be:
+
+    /packit test --identifier test-1
+
+That command will execute the job with this specific identifier, nothing else.
+
+And what if someone wants to execute more than one job? Users can easily use multiple identifiers in a comma-separated list:
+
+    /packit test --identifier test-1,test-2
```

Comment:
I would maybe use real names so people can better understand the usecase.

E.g. `smoke` and `full`,...

Reply:
I like how you use real names in the following section.